nabu-comms.md

```diff
@@ -49,14 +49,27 @@ messages until the NABU requests to see them.
 All messages exchanged use the little-endian byte order.  Thus, a 16
 bit number 0xaa00 is transmitted as the two bytes 0x00 and 0xaa.
 
-Each message consists of a 16 bit length field followed by the number
-of bytes indicated by the length field.  The maximum length of a
-message is 32767 bytes to ensure that the most significant bit of the
-message length is never set.  This helps in crash recovery (see
-below).
-
-No further framing is provided.  Messages must be completely
-transmitted within one second.  The receiver should ignore partial
+Messages must be completely transmitted within one second, intended to
+reduce error recovery time.
+
+Each message consists of a 16 bit length field followed by the number of
+bytes indicated by the length field.  Because messages must be completely
+transmitted within one second, there is a hard ceiling on the maximum
+transmission unit, dictacted by the NABU hardware.  With the native
+tramission rate of 111860 bits per second and many network adapter
+implementations using 2 stop bits, 11186 bytes is the practical limit
```

Comment:
I believe this number is even lower, 10,169. 1 start bit, 8 data bits, and 2 stop bits is 11 bits. So the maximum throughput in real data is 8 out of 11 bits or 72.72% of 111860bps which is the same as 111890bps/11b=10169Bps.

Reply:
I always forget the START bit.  Thanks!

Reply:
Everybody needs a start bit, but nobody remembers them 😄